internal/services/compute/managed_disk_resource.go

```diff
@@ -145,6 +145,20 @@ func resourceManagedDisk() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"disk_iops_read_only": {
```

Comment:
can you set both this and `disk_mbps_read_write`?

Reply:
mbps_rw and iops_rw is already present, while missing a ValidateFunc, it is added here: https://github.com/hashicorp/terraform-provider-azurerm/pull/14028

Reply:
sorry what i mean is can you set `disk_iops_read_only` and `disk_iops_read_write` at the same time? when is one property used vs the other?

Reply:
oh I see. They can be set at same time, when it is an ultra disk with max_shares greater than 1.
read_write is for normal case, and read_only is  used when the vm is only configured the read access to the disk.

Here is the screenshot where I captured from Azure portal:
![image](https://user-images.githubusercontent.com/10579712/140671558-4768924b-3c40-4b2f-bce4-11ea3ab4ff92.png)
And this section has some examples: https://docs.microsoft.com/en-us/azure/virtual-machines/disks-shared#ultra-disk-performance-throttles